.travis.yml

```diff
@@ -1,23 +1,30 @@
 language: python
 python:
-  - "2.7"
+- '2.7'
 addons:
   apt:
     packages:
-      - swig
-      - libusb-1.0-0-dev
-      - libprotobuf-dev
-      - protobuf-compiler
+    - swig
```

Comment:
why did you dedent all of these lists? It's actually a little confusing now (keys of a dict and lists for a key are at the same level now)